.github/workflows/issues.yml

```diff
@@ -0,0 +1,19 @@
+name: Add issue or pull request to Project
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add issue to project
+        uses: actions/add-to-project@v0.5.0
+        with:
+          project-url: https://github.com/orgs/shipwright-io/projects/6
+          github-token: ${{ secrets.ADD_TO_PROJECT_TOKEN }}
```

Comment:
For me to learn, this is not a GitHub default token, correct ? We would need to configure it in all the repositories.

Reply:
Correct, we need to create it, and set it up as a secret at the organization level, not for each individual repository.

Reply:
We added the secret, fyi.